interface/ceed-basis.c

```diff
@@ -729,12 +729,22 @@ int CeedBasisReferenceCopy(CeedBasis basis, CeedBasis *basis_copy) {
 int CeedBasisView(CeedBasis basis, FILE *stream) {
   int ierr;
   CeedFESpace FE_space = basis->basis_space;
+  CeedElemTopology topo = basis->topo;
+  // Print FE space and element topology of the basis
+  if (basis->tensor_basis) {
+    fprintf(stream, "CeedBasis (%s on a %s element): dim=%d P=%d Q=%d\n",
+            CeedFESpaces[FE_space], CeedElemTopologies[topo],
+            basis->dim, basis->P_1d, basis->Q_1d);
+  } else {
+    fprintf(stream, "CeedBasis (%s on a %s element): dim=%d P=%d Q=%d\n",
+            CeedFESpaces[FE_space], CeedElemTopologies[topo],
+            basis->dim, basis->P, basis->Q);
+  }
+  // Print quadrature data, interpolation/gradient/divergene/curl of the basis
```

Comment:
@jeremylt how about that?

Reply:
You'll need to change some .out files, but it looks good to me

Reply:
I'll do. Thanks